docs/compose/Text/README.md

```diff
@@ -0,0 +1,45 @@
+# Text
+
+## Installation
+
+Backpack Compose is available through [Maven Central](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-compose). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
+
+## Usage
+
+Example of a text composable with `label1` style and secondary text color
+
+```Kotlin
+import androidx.compose.runtime.Composable
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.theme.BpkTheme
+
+@Composable
+fun FlightText() {
```

Comment:
I think we can just have a sample of the component without the wrapper function :)

Reply:
yeah i was wondering about that - i thought it might be a little clearer, but it's probably repetitive

Reply:
Yeah, it should be clear that you need to use @ Composable functions for the components - so no need to have a wrapper for each sample :)